tools/integration_tests/mounting/gcsfuse_test.go

```diff
@@ -551,3 +558,69 @@ func (t *GcsfuseTest) HelpFlags() {
 		ExpectEq(nil, err, "case %d\nOutput:\n%s", i, output)
 	}
 }
+
+func (t *GcsfuseTest) RelativeLogFilePath() {
+	// Create a temp file in current working directory
+	currentDirTestFile := filepath.Join(t.dir, "test.txt")
+	_, err := os.Create(currentDirTestFile)
+	AssertEq(nil, err)
+	defer os.Remove(currentDirTestFile)
+
+	// Create a temp log file in home directory
+	homeDir, err := os.UserHomeDir()
+	AssertEq(nil, err)
+
+	homeTestFile := filepath.Join(homeDir, "test_home.json")
+	_, err = os.Create(homeTestFile)
+	defer os.Remove(homeTestFile)
+
+	// Specify log file in different way.
```

Comment:
you can mention without --foreground flag here itself instead of on every entry in the below array

Reply:
done.